RomanASP.py

```diff
@@ -433,16 +433,26 @@ def main():
             if use_real_images:
                 detections = exposures[np.where(exposures['DETECTED'])]
                 parq_file = find_parq(ID, path = sn_path)
-                df = open_parq(parq_file, path = sn_path)
+                if len(str(ID)) != 11:
```

Comment:
Both some question as above about the `11`, but also a structural question.  Why does this check have to be repeated here.  Can't the code just be the same?